api/envoy/extensions/http/stateful_session/header/v3/header.proto

```diff
@@ -24,19 +24,48 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // <envoy_v3_api_msg_extensions.filters.http.stateful_session.v3.StatefulSession>`.
 //
 // For example, if the header name is set to ``session-header``, Envoy will prefer ``1.2.3.4:80``
-// as the upstream host when the request contains the following header:
+// as the upstream host when the request contains the following header (Note that plain text is used
+// here as example but actually header value will be base64 encoded):
 //
 // .. code-block:: none
 //
-//     session-header: "MS4yLjMuNDo4MA=="
+//     session-header: "1.2.3.4:80"
```

Comment:
nit: keep the base64 value, and add a comment below that says `base64("1.2.3.4:80") => "MS4yLjMuNDo4MA=="`.